docs/package-lock.json

```diff
@@ -0,0 +1,3015 @@
+{
```

Comment:
@vchepeli  I don't know ;)
I didn't because of the message 'created a lockfile as package-lock.json. You should commit this file.' And there's https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5
but on the other hand it isn't required by me so I can do as you suggest, if that makes the merge easier?
It's an artifact of running following in docs directory, which is required to validate local-site.yml: 
npm i @antora/site-generator-default